beanstalk_worker/worker_dockerfile_lint.py

```diff
@@ -54,24 +54,46 @@ def lint_job_data(job_data):
          "registry.centos.org/pipeline-images/dockerfile-lint"],
         stdout=subprocess.PIPE
     ).communicate()
-    logger.log(level=logging.INFO, msg="Dockerfile Lint check done")
-
-    response = {
-        "logs": out,
-        "action": "notify_user",
-        "namespace": namespace
-    }
 
-    bs.use("master_tube")
-    jid = bs.put(json.dumps(response))
-    logger.log(
-        level=logging.INFO,
-        msg="Put job on 'master_tube' tube with id: %d" % jid
-    )
+    if err is None:
+        logger.log(level=logging.INFO, msg="Dockerfile Lint check done")
+        response = {
+            "logs": out,
+            "linter_results": True,
+            "action": "notify_user",
+            "namespace": job_data.get('namespace'),
+            "notify_email": notify_email,
```

Comment:
if notify_email is not used anywhere, you can use, `job_data.get("notify_email")` here